src/components/Thanks/CommentAsk.vue

```diff
@@ -0,0 +1,183 @@
+							Tell the borrower the reasons they should believe in themselves.
+						</li><li>
+							Chime in on a comment another lender has written.
+						</li>
+					</ul>
+					<h3>Don't</h3>
+					<ul class="tw-mt-0">
+						<li>Don't write comments about Kiva itself. Don't write comments about issues relating to historical repayments from the lending partner. Keep it on topic.</li>
+						<li>
+							Don't attack, harass or threaten anyone.
+						</li><li>
+							Don't spam our lenders.
+						</li>
+					</ul>
+					<p>
+						Your rights and responsibilities as part of the Kiva community are laid out in the <router-link
+							to="/legal/terms"
+							target="_blank"
+						>
+							Terms of Use
+						</router-link> and the following guidelines, which apply to lending teams, comments, conversation tab posts, private messages, profiles and any other content posted on Kiva or Kiva's social media properties.
+					</p>
+					<p>
+						The Kiva team reserves the right to remove posts and messages at our sole discretion. Frequent guideline violations may also lead to the loss of posting privileges. If you have any questions, concerns or grievances related to these guidelines, please write to our team at <a href="mailto:contactus@kiva.org">contactus@kiva.org</a>.
+					</p>
+				<!-- eslint-enable max-len -->
+				</div>
+			</kv-lightbox>
+		</div>
+	</transition>
+</template>
+
+<script>
+import { gql } from '@apollo/client';
+
+import { mdiPencilOutline } from '@mdi/js';
+import KvMaterialIcon from '~/@kiva/kv-components/vue/KvMaterialIcon';
+import KvButton from '~/@kiva/kv-components/vue/KvButton';
+import KvLightbox from '~/@kiva/kv-components/vue/KvLightbox';
+
+export default {
+	name: 'CommentAsk',
+	inject: ['apollo', 'cookieStore'],
+	components: {
+		KvMaterialIcon,
+		KvButton,
+		KvLightbox,
+	},
+	props: {
+		loanName: {
+			type: String,
+			default: ''
+		},
+		loanId: {
+			type: Number,
+			default: null
+		},
+	},
+	data() {
+		return {
+			mdiPencilOutline,
+			showComments: true,
+			loading: false,
+			showLightbox: false,
+		};
+	},
+	computed: {
+		defaultComment() {
+			return `This loan not only helps ${this.loanName}, but helps their community as well!`;
+		},
+		buttonState() {
+			if (this.loading) return 'loading';
+			return '';
+		},
+	},
+	methods: {
+		submitComment() {
+			this.loading = true;
+			this.apollo.mutate({
+				mutation: gql`mutation commentOnLoan($id: Int!, $body: String) {
+					loan(id: $id) {
+						addComment(body: $body)
+					}
+				}`,
+				variables: {
+					id: this.loanId,
+					body: this.defaultComment
+				}
+			}).then(() => {
+				this.$showTipMsg(`Thank you for helping ${this.loanName}!`);
+				this.showComments = false;
+			}).catch(e => {
+				console.error(e);
+				this.$showTipMsg('There was a problem commenting on this loan', 'error');
+			}).finally(() => {
+				this.loading = false;
+			});
+		},
+	},
+	mounted() {
+	},
+};
+
+</script>
+
+<style lang="scss" scoped>
```

Comment:
Remove?

Reply:
I think I still need both these imports since I did not refactor the grid layout and I'm still using the foundation row/column classes